### PR TITLE
Setup Factory Girl

### PR DESCRIPTION
**Pull request by CorainChicago**
This setups factory girl in the app with a factory established for accommodations

closes Issue #31

### CODE REVIEW

File: spec/fixtures/volunteers.yml [hunk 1]
```diff
@@ -2,12 +2,10 @@
```
Comment (BintLopez) [inline spec/fixtures/volunteers.yml]: Feel free to get rid of the fixtures :) especially since we're using factories now

**Comment by BintLopez** [comment]
@CorainChicago -- many thanks for setting this up!!!! You rock!

File: db/schema.rb [hunk 2]
```diff
@@ -120,7 +120,7 @@
 
   create_table "volunteers", force: :cascade do |t|
     t.string   "gender"
-    t.string   "capacity"
+    t.string   "type"
```
Comment (BintLopez) [inline db/schema.rb]: Why the change to schema with no corresponding migration? Wondering if you pulled upstream master into your branched and ran migrations? Doing that should resolve this diff methinks.

File: spec/fixtures/volunteers.yml (hunk 1 above)
Comment (CorainChicago) [inline spec/fixtures/volunteers.yml]: I'll delete the fixtures and make factories for them.

File: spec/factories.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
+FactoryGirl.define do
+
```
Comment (CorainChicago) [inline spec/factories.rb]: I started to question creating all the attributes with Faker for an object that were in the fixtures file and started to consider reducing the set attributes to the ones required to make a valid object.  Deleting is easy so I left them in.
